### ISSUE TITLE
Select isGrouped - remove aria-label from example

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-react#1738]
Currently, aria-label is defined on the example labeled **Grouped Checkbox Select Input**, but this isn't actually used by this variation of the component. This should be removed from the example.
